tsconfig.json

```diff
@@ -1,6 +1 @@
-{
```

Comment:
this was giving my IDE an error and didn't seem like it was doing anything, so I made this empty